src/test/regress/sql/multi_large_table_join_planning.sql

```diff
@@ -36,13 +36,12 @@ WHERE
 	l_partkey = p_partkey AND
 	c_custkey = o_custkey AND
         (l_quantity > 5.0 OR l_extendedprice > 1200.0) AND
-        p_size > 8 AND o_totalprice > 10 AND
-        c_acctbal < 5000.0
+        p_size > 8 AND o_totalprice > 10.0 AND
+        c_acctbal < 5000.0 AND l_partkey < 1000
```

Comment:
yes. limit 30 was displayed as
'30'::bigint in pg95. where pg94 displayed 30:bigint. I could not find a way to make them output the same text.